Mw/fix funder sort order by

michaelwood opened the pull request:
Fixes sorting by funder. Copy change TBC `Total GBP Amount - Highest First"` to `Total GBP Amount to Organisations`. 

<img width="1134" height="423" alt="image" src="https://github.com/user-attachments/assets/2077c9c0-6a3f-468b-9f80-fa9cab149ab1" />

and 

<img width="1134" height="423" alt="image" src="https://github.com/user-attachments/assets/998dd93b-1e2f-4bd2-9e6b-7f2c6bd09857" />

Requires a data reload.

Commit:
import/funder view: Fix order-by amount due to field location change

Also update copy to make sure it is obvious this is the amount only for
organisations.

Fixes: https://github.com/ThreeSixtyGiving/grantnav/issues/1203

Commit:
frontend: When development offline downloading the schema can be optional

mariongalley reviewed (dismissed):
Happy with the copy change. Haven't had the opportunity to test but if you're satisfied the sort order works and no regressions have been introduced then I'm OK to approve.

Commit:
tests: Add test to check the ordering of funders applies correctly

Check that ordering for funders on search results is correctly applied

Commit:
tests: data: Update funders.jsonl updated aggregate data structure

Commit:
templates: funders-search: Remove debug output

Commit:
tests: browser_test_case: Add option to easily re-attach browser head

As the default is best kept as headless add another env to attach it if
needed (e.g. debugging tests)

Commit:
tests: browser: Reduce the timeout for search pages to finish loading

No one is going to wait 5 minutes! Reduce to 2.

michaelwood commented:
Bypassing rules - as no one available for code review and this is currently broken in live.
